app/routes/overview/components/CompactEvents.js

```diff
@@ -36,6 +37,11 @@ export default class CompactEvents extends Component<Props> {
               <i className="fa fa-circle" />
             </span>
             <Link to={`/events/${event.id}`}>{event.title}</Link>
+            {event.pinned && (
+              <Tooltip content="Dette arrangementet er festet til forsiden">
+                <i className="fa fa-thumb-tack" />
```

Comment:
How about some quick `CSS` magic ? 💯 
You can give it some `px` margin, and a quick transform, maybe slap on some color 🔥 

```js
style={{
  transform: 'rotate(-20deg)',
  marginRight: '4px',
  color: '#BE1600'
}}
```
 
![image](https://user-images.githubusercontent.com/23152018/54167331-7ebbc300-4469-11e9-9c8f-f4f7fd25cbd9.png)

Reply:
Yeah that works out great!